x-pack/solutions/observability/packages/utils_server/entities/get_data_streams_for_entity.ts

```diff
@@ -54,10 +54,26 @@ export async function getDataStreamsForEntity({
   });
 
   const dataStreams = uniq(
-    compact(await resolveIndexResponse.indices.flatMap((idx) => idx.data_stream))
+    compact(
+      await resolveIndexResponse.indices.flatMap((idx) => {
+        const remoteCluster = extractRemoteCluster(idx.name);
+        if (remoteCluster) {
+          return `${remoteCluster}:${idx.data_stream}`;
+        }
+        return idx.data_stream;
+      })
+    )
   );
 
   return {
     dataStreams,
   };
 }
+
+const extractRemoteCluster = (index: string) => {
+  const match = index.match(/^([^:]+):/);
```

Comment:
this seems like an overly complex regex for checking the presence of a `:` no? we can use `.includes(':') ? index.split(':')[0] : null`